src/analysis/functions.rs

```diff
@@ -224,6 +224,21 @@ fn fixup_special_functions(
     }
 }
 
+fn find_callback_bound_to_destructor(
+    callbacks: &[Trampoline],
+    destroy: &mut Trampoline,
+    destroy_index: usize,
+) -> bool {
+    for call in callbacks.iter() {
```

Comment:
Seems `.iter()` not needed

Reply:
Maybe change to `let Some(call) = callbacks.iter().find(...) {...}` but not sure that it better

Reply:
I prefer the `for` loop syntax so I just removed the `.iter()` call.

Reply:
I too 😉